client/src/components/DebouncedTextInput/DebouncedTextInput.stories.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+import { DebouncedTextInput } from "./DebouncedTextInput.js";
+
+export default {
+  title: "Input/DebouncedTextInput",
+  component: DebouncedTextInput,
+};
+
+const Template = (args) => <DebouncedTextInput {...args} />;
+
+export const Basic = Template.bind({});
+Basic.args = {
+  placeHolder: "Write something",
+};
```

Comment:
Lots of unused props on this one. A lot of them, like `a11y_label` have defaults or are optional, but I think at least having controls for them would still be worthwhile.